ods_ci/tests/Tests/1100__data_science_pipelines/1100__data-science-pipelines-kfp.robot

```diff
@@ -95,8 +95,19 @@ End To End Pipeline Workflow Using Kfp
     IF    ${ray} == ${TRUE}
         Setup Kueue Resources    ${project}    cluster-queue-user    resource-flavor-user    local-queue-user
     END
-    ${run_id}    Create Run From Pipeline Func    ${username}    ${password}    ${project}
-    ...    ${python_file}    ${method_name}    pipeline_params=${pipeline_params}
+    # The run_robot_test.sh is sending the --variablefile ${TEST_VARIABLES_FILE} which may contain the `PIP_INDEX_URL`
+    # and `PIP_TRUSTED_HOST` variables, e.g. for disconnected testing. In such case there is no way to detect if
+    # `PIP_INDEX_URL` or `PIP_TRUSTED_HOST was definied. The only way is by calling Run Keyword And Return Status
+    # Log If it returns true, means that we are running the test in a disconnected environment.
+    ${pip_index_url_definied}    Run Keyword And Return Status    Log    ${PIP_INDEX_URL}
```

Comment:
that is not the only way, but there is a built-in function in RF: https://robotframework.org/robotframework/latest/libraries/BuiltIn.html#Get%20Variable%20Value

Reply:
It will end up in the same situation. Logging or getting will return an error if the variable doesn't exists.
The real way to solve is by checking if the variable exists and I didn't find any function.